inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -293,6 +305,30 @@ public InlongGroupInfo get(String groupId, UserInfo opInfo) {
         return groupInfo;
     }
 
+    @Override
+    public String getTenant(String groupId, String operator) {
+        InlongGroupEntity groupEntity = groupMapper.selectByGroupIdWithoutTenant(groupId);
+        String tenant = groupEntity.getTenant();
+        if (Objects.equals(InlongConstants.DEFAULT_PULSAR_TENANT, tenant)) {
+            return tenant;
+        }
+        InlongTenantInfo tenantInfo = tenantService.getByName(tenant);
+        if (tenantInfo == null) {
+            String errMsg = String.format("tenant=[%s] not found", tenant);
+            LOGGER.error(errMsg);
+            throw new BusinessException(errMsg);
+        }
+
+        InlongRoleInfo inlongRoleInfo = inlongRoleService.getByUsername(operator);
+        TenantUserRoleEntity tenantRoleInfo = tenantUserRoleEntityMapper.selectByUsernameAndTenant(operator, tenant);
+        if (inlongRoleInfo == null && tenantRoleInfo == null) {
```

Comment:
what if only `inlongRoleInfo` or `tenantRoleInfo`  is null?

Reply:
`inlongRoleInfo` and `tenantRoleInfo` is null.Because when there is no group tenant permission, the corresponding tenant information cannot be obtained。